specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,294 @@
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               }
+            ]
+         }
+      },
+      "/{scope}/providers/Microsoft.Subscription/createSubscription":{  
+         "post":{  
+            "description":"Creates an Azure subscription",
+            "operationId":"subscriptionFactory_createSubscription",
+            "x-ms-long-running-operation":true,
+            "responses":{  
+               "200":{  
+                  "description":"Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationResult"
+                  }
+               },
+               "202":{  
+                  "description":"Accepted. Subscription creation is in progress.",
+                  "headers":{  
+                     "Location":{  
+                        "description":"GET this URL to retrieve the status of the asynchronous operation.",
+                        "type":"string"
+                     },
+                     "Retry-After":{  
+                        "description":"The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                        "type":"string"
+                     }
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "name":"scope",
+                  "in":"path",
+                  "description":"The scope of the subscription creation. Valid scopes include: providers/Microsoft.Billing/billingAccounts/{accountId}",
+                  "required":true,
+                  "type":"string"
+               },
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"body",
+                  "in":"body",
+                  "description":"The subscription creation parameters.",
+                  "required":true,
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationParameters"
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "definitions":{  
+      "SubscriptionCreationResult":{  
+         "properties":{  
+            "subscriptionLink":{  
+               "type":"string"
+            }
+         }
+      },
+      "SubscriptionCreationParameters":{  
+         "properties":{  
+            "displayName":{  
+               "type":"string"
+            },
+            "owners":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/AdPrincipal"
+               },
+               "description":"An array of Active Directory Principals"
```

Comment:
"The list of principals that should be granted Owner access on the subscription."